src/decaylanguage/decay/decay.py

```diff
@@ -505,6 +507,140 @@ def _build_decay_modes(
             decay_modes[mother] = DecayMode.from_dict(d)
 
 
+def _expand_decay_modes(
+    decay_chain: DecayChainDict,
+    *,
+    top: bool = True,
+    aliases: dict[str, str] | None = None,
+) -> list[str]:
+    """Given a dict with 1 key (the mother particle) whose value is a list of
+    decay modes, recursively replace all decay modes with decay descriptors.
+
+    Parameters
+    ----------
+    decay_chain: dict
+    top: bool, optional, default=True
+    aliases: dict[str, str], optional, default={}
```

Comment:
aliases are useful. This is a reason to expand the parameter descriptions a bit more and maybe use aliases in one of the nice examples below?